libraries/websphere_base.rb

```diff
@@ -50,7 +50,7 @@ def manageprofiles_exec(cmd, options)
           user new_resource.run_user
           group new_resource.run_group
           command command
-          sensitive new_resource.sensitive_exec
+          sensitive false #new_resource.sensitive_exec
```

Comment:
Sandeep you don't need to do this as you can still get the output from `stacktrace.out`